### PR TITLE
new examples

### PR DESCRIPTION
**Pull request by soniamitchell**
* globbing in read
* globbing in write

### CODE REVIEW

File: content/docs/interface/example0/_index.md [hunk 1]
```diff
@@ -1105,3 +1105,221 @@ write:
...
+- data_product: real/data/e8d7af00c8f8e24c2790e2a32241bc1bfc8cf011/*
+  description: A csv file
+  file_type: csv
+  use:
+    version: 1.0.0
+- data_product: real/data/e8d7af00c8f8e24c2790e2a32241bc1bfc8cf011/thing/1
+  description: A csv file
+  file_type: csv
+  use:
+    version: 1.0.1
```
Comment (richardreeve) [inline content/docs/interface/example0/_index.md:1319]: A major version bump just goes to the next whole number:
```suggestion
    version: 1.0.0
```

File: content/docs/interface/example0/_index.md [hunk 2]
```diff
@@ -1105,3 +1105,221 @@ write:
...
+- data_product: real/data/e8d7af00c8f8e24c2790e2a32241bc1bfc8cf011/1
+  description: A csv file
+  file_type: csv
+  use:
+    version: 1.0.1
```
Comment (richardreeve) [inline content/docs/interface/example0/_index.md:1324]: Ditto:
```suggestion
    version: 1.0.0
```

File: content/docs/interface/example0/_index.md [hunk 3]
```diff
@@ -1105,3 +1105,221 @@ write:
...
+#### User written *config.yaml*
+
+```yaml
+run_metadata:
+  description: Register a file in the pipeline
+  local_data_registry_url: http://localhost:8000/api/
+  remote_data_registry_url: https://data.scrc.uk/api/
+  default_input_namespace: username
+  default_output_namespace: username
+  write_data_store: test_datastore/
```
Comment (richardreeve) [inline content/docs/interface/example0/_index.md:1124]: Shouldn't a `write_data_store` be an absolute path? You have it quite a bit in this PR, but I'm not confident I know what it's relative to...

**Review by richardreeve — CHANGES_REQUESTED** [review]
Minor problems with version numbers.

File: content/docs/interface/example0/_index.md (hunk 3 above)
Comment (soniamitchell) [reply content/docs/interface/example0/_index.md:1124]: sure, that makes sense

File: content/docs/interface/example0/_index.md (hunk 1 above)
Comment (soniamitchell) [reply content/docs/interface/example0/_index.md:1319]: oh no! I'll need to fix my code 😅

**Review by richardreeve — APPROVED** [review]

**Pull request merged**